blobfuse/include/BlobfuseGcCache.h

```diff
@@ -20,7 +21,7 @@ class gc_cache
         file_cache_timeout_in_seconds(timeout),
         m_current_usage(0){}
     void run();
-    void uncache_file(std::string path);
+    void uncache_file(std::string path, bool force = false);
```

Comment:
will the force work when the file is open?

Reply:
On close we just submit the file to gc-cache for eviction. If the force flag is truned on its submitted to the front of the list to give it high priority. Rest of the logic for eviction remains same. If file is open or we fail to take exclusive lock it can not be evicted.